src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.dto.CartDto;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/cart")
```

Comment:
I suggest it will be better to change to plural form "carts" as in other controllers